website/docs/getting-started/insecure-registry.md

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_position: 10
+title: Adding an insecure registry
+description: Modifying Podman for an insecure registry
+keywords: [podman desktop, podman, containers, registry, podman machine]
+tags: [pods]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Adding an Insecure Registry to Podman Desktop
+
+In this guide, you will learn how to add an insecure registry to Podman Desktop. An insecure registry allows you to pull and push container images either over an unencrypted HTTP connection, or a HTTPS connection with a self-signed unverified certificate. Please note that using an insecure registry can expose your data to security risks, so it's recommended to use secure connections with a verifiable certificate whenever possible.
+
+#### Prerequisites
+
+- Podman installed and running (VM if on macOS or Windows).
+
+#### Procedure
+
+<Tabs groupId="operating-systems">
+<TabItem value="macwin" label="macOS & Windows">
+
+1. Add your insecure registry within Settings -> Registries.
```

Comment:
I think @themr0c is using `**<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**`
**<icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**